### PR TITLE
docs(retry): fix rendering docs

### PR DESCRIPTION
**Pull request by jakovljevic-mladen**
**Description:**
Please check the related issue for more info. This is how the page looks like with this fix (sorry for the large image):

image

**Related issue (if exists):**
#6660

### CODE REVIEW

File: src/internal/operators/retry.ts [hunk 1]
```diff
@@ -63,22 +76,10 @@ export interface RetryConfig {
  *
  * @see {@link retryWhen}
  *
- * @param count - Number of retry attempts before failing.
- * @param resetOnSuccess - When set to `true` every successful emission will reset the error count
+ * @param configOrCount - Either number of retry attempts before failing or a {@link RetryConfig} object.
  * @return A function that returns an Observable that will resubscribe to the
  * source stream when the source stream errors, at most `count` times.
  */
-export function retry<T>(count?: number): MonoTypeOperatorFunction<T>;
```
Comment (jakovljevic-mladen) [inline src/internal/operators/retry.ts:71]: Moved [here](https://github.com/ReactiveX/rxjs/pull/6850/files#diff-487ea11567bb6ea30a20b6e8431d91f13374a9345f7fd3b5dbc431273e099ab0R34).

File: src/internal/operators/retry.ts [hunk 2]
```diff
@@ -63,22 +76,10 @@ export interface RetryConfig {
...
-
-/**
- * Returns an observable that mirrors the source observable unless it errors. If it errors, the source observable
- * will be resubscribed to (or "retried") based on the configuration passed here. See documentation
- * for {@link RetryConfig} for more details.
- *
- * @param config - The retry configuration
- */
-export function retry<T>(config: RetryConfig): MonoTypeOperatorFunction<T>;
```
Comment (jakovljevic-mladen) [inline src/internal/operators/retry.ts:80]: Moved [here](https://github.com/ReactiveX/rxjs/pull/6850/files#diff-487ea11567bb6ea30a20b6e8431d91f13374a9345f7fd3b5dbc431273e099ab0R35).

File: src/internal/operators/retry.ts [hunk 3]
```diff
@@ -63,22 +76,10 @@ export interface RetryConfig {
...
- * @param resetOnSuccess - When set to `true` every successful emission will reset the error count
+ * @param configOrCount - Either number of retry attempts before failing or a {@link RetryConfig} object.
  * @return A function that returns an Observable that will resubscribe to the
  * source stream when the source stream errors, at most `count` times.
  */
-export function retry<T>(count?: number): MonoTypeOperatorFunction<T>;
-
-/**
- * Returns an observable that mirrors the source observable unless it errors. If it errors, the source observable
- * will be resubscribed to (or "retried") based on the configuration passed here. See documentation
```
Comment (jakovljevic-mladen) [inline src/internal/operators/retry.ts:75]: Moved [here](https://github.com/ReactiveX/rxjs/pull/6850/files#diff-487ea11567bb6ea30a20b6e8431d91f13374a9345f7fd3b5dbc431273e099ab0R45-R47).

File: src/internal/operators/retry.ts [hunk 4]
```diff
@@ -63,22 +76,10 @@ export interface RetryConfig {
  *
  * @see {@link retryWhen}
  *
- * @param count - Number of retry attempts before failing.
- * @param resetOnSuccess - When set to `true` every successful emission will reset the error count
```
Comment (jakovljevic-mladen) [inline src/internal/operators/retry.ts:67]: This parameter doesn't exist anymore in this form, so I removed the docs for it. It is now part of the RetryConfig and it's properly documented.

**Review by benlesh — APPROVED** [review]

**Pull request merged**